capsules/src/net/util.rs

```diff
@@ -0,0 +1,101 @@
+/// Utility functions used in the 6LoWPAN implementation
+
+/// Verifies that all bits beyond prefix_len in the slice are zero
+pub fn verify_prefix_len(prefix: &[u8], prefix_len: u8) -> bool {
```

Comment:
I think this is just

```rust
prefix.iter().skip(prefix_len / 8).all(|&b| b == 0)
```

Maybe. It's totally clear what prefix_len is meant to be... is it the number of bits? What is the `prefix_len & 0x7 != 0` doing?